packages/tabs/src/TabTitle.tsx

```diff
@@ -113,6 +113,23 @@ const listTitle = css`
   }
 `;
 
+function useDocumentActiveElement() {
+  const [activeEl, setActiveEl] = React.useState<Element | null>(null);
+
+  const handleFocusIn = () => {
+    setActiveEl(document.activeElement);
+  };
```

Comment:
`useCallback`

Reply:
👍

Reply:
> > Hm, I'm not seeing that locally
> 
> Seems to only happen in Firefox

I'm not seeing it broken in Firefox either